packages/core/src/render3/util.ts

```diff
@@ -291,4 +291,18 @@ export function resolveDocument(element: RElement & {ownerDocument: Document}) {
 
 export function resolveBody(element: RElement & {ownerDocument: Document}) {
   return {name: 'body', target: element.ownerDocument.body};
-}
\ No newline at end of file
+}
+
+/**
+ * The special delimiter we use to separate property names, prefixes, and suffixes
+ * in property binding metadata. See storeBindingMetadata().
+ */
+export const INTERPOLATION_DELIMITER = `�`;
+
+/**
+ * Returns whether or not the given string is a property metadata string.
+ * See storeBindingMetadata().
+ */
+export function isPropMetadataString(str: string): boolean {
+  return new RegExp(/�/).test(str);
```

Comment:
```suggestion
  return str.indexOf(INTERPOLATION_DELIMITER) >= 0;
```

Should be more efficient.